asdf/tests/test_util.py

```diff
@@ -99,3 +99,20 @@ def read(self, size=-1):
     fd = generic_io.get_file(OnlyHasAReadMethod(content))
     assert util.get_file_type(fd) == expected_type
     assert fd.read() == content
+
+
+def test_minversion():
+    import numpy
+    import yaml
+
+    good_versions = ["1.16", "1.16.1", "1.16.0.dev", "1.16dev"]
+    bad_versions = ["100000", "100000.2rc1"]
+    for version in good_versions:
+        assert util.minversion(numpy, version)
```

Comment:
Will this fail for numpy versions <1.16? The pyproject.toml dependencies have numpy >= 1.10. I'm failing to build any old numpy (<1.16) locally so haven't so far been able to test this.

Reply:
Oh numpy 1.18 and below isn't even supported by numpy anymore. So we may just want to pin above that anyways.